pkg/ccl/sqlproxyccl/acl/file_test.go

```diff
@@ -358,6 +371,9 @@ func TestAllowListFileParsing(t *testing.T) {
 			}},
 		}
 		for _, tc := range cases {
+			defer leaktest.AfterTest(t)()
+			testutilsccl.ServerlessOnly(t)
```

Comment:
Same as above - moving this outside the for loop.

Reply:
Fixed